cockatrice/src/tab_deck_editor.cpp

```diff
@@ -251,6 +255,7 @@ void TabDeckEditor::createMenus()
     deckMenu->addAction(aNewDeck);
     deckMenu->addAction(aLoadDeck);
     deckMenu->addAction(aSaveDeck);
+    deckMenu->addAction(aExportDeck);
```

Comment:
Maybe we can put this into `Deck Editor -> Analyze Deck Online` ?

Reply:
I don't think it's a great fit within there.

But what about placing it close to it?

```
--------------------------------
Print deck...
Export deck to decklist.org
Analyze deck online           >
--------------------------------
```

Alternatively we could restructure + rename that menu and then have the 3 webpages there:
 - `Send deck to online service` 
      - `Create decklist (decklist.org)`
      - `Analyze deck (deckstats.net)`
      - `Analyze deck (tappedout.net)`

Reply:
I very much like that idea, it's implemented in the latest commit.